assets/src/sass/blocks/_carousel.scss

```diff
@@ -0,0 +1,79 @@
+@import "../../../../node_modules/@splidejs/splide/src/css/core";
```

Comment:
Nah, the theme styles here are still being built in gulp, and there's no import path handler for node modules. (As an aside, it'd be really nice to rewrite the old gulp build tasks so that at least the JS and CSS parts of it are handled in webpack like the newer parts of the theme codebase...)